tests/test_validation.py

```diff
@@ -94,3 +99,58 @@ def test_validate(model, val_sat_zarr_path, mocker):
         num_workers=0,
         batch_limit=4,
     )
+
+
+def test_validate_cli(val_sat_zarr_path, mocker):
+    # write out an example model.py file
+    with open("model.py", "w") as f:
+        f.write(
+            """
+from cloudcasting.models import AbstractModel
```

Comment:
This is totally a style preference, so I don't feel strongly about this. But I think it would be nicer to just move these file strings to global variables within this file rather than have them unindented like this inside the test function

Reply:
Yeah, this is annoying. But I suspect they'll be even more clutter at the top of the file, and also aren't shared across tests (else they may as well be in `conftest.py`). So probably will just leave the weirdness.